packages/ui/react-ui-editor/src/components/Toolbar/Toolbar.tsx

```diff
@@ -135,7 +134,19 @@ const ToolbarButton = ({ Icon, children, ...props }: ToolbarButtonProps) => {
   );
 };
 
-const ToolbarSeparator = () => <div role='separator' className='grow' />;
+//
+// Heading
+//
+
+const HeadingIcons: { [key: string]: Icon } = {
+  '0': Paragraph,
+  '1': TextHOne,
+  '2': TextHTwo,
+  '3': TextHThree,
+  '4': TextHFour,
+  '5': TextHFive,
+  '6': TextHSix,
```

Comment:
See my other remark about using strings instead of importing and passing around components.